web/client/src/components/form/TokenForm.jsx

```diff
@@ -144,19 +144,7 @@ export default function TokenForm(props) {
     let finalCredentials;
     if (useUploadedCredential) {
       const tokenCred = JSON.parse(values.tokenCredentials);
-      if (
-        tokenCred["web"] !== undefined &&
-        tokenCred["installed"] === undefined
-      ) {
-        finalCredentials = tokenCred["web"];
-      } else if (
-        tokenCred["web"] === undefined &&
-        tokenCred["installed"] !== undefined
-      ) {
-        finalCredentials = tokenCred["installed"];
-      } else {
-        finalCredentials = tokenCred;
-      }
+      finalCredentials = tokenCred;
```

Comment:
Instead of doing this, how about we set `const finalCredentials = JSON.parse(values.tokenCredentials)`. Also, since we are no longer breaking down the `tokenCred` to a `finalCred`, maybe we can rename `finalCred` to `tokenCred`